lib/test_bot.rb

```diff
@@ -146,17 +148,17 @@ def test_ci_upload(tap)
       if pr = ENV["UPSTREAM_PULL_REQUEST"]
         if Homebrew.args.dry_run?
           puts <<~EOS
-            git am --abort
-            git rebase --abort
-            git checkout -f master
-            git reset --hard origin/master
+            #{GIT} am --abort
+            #{GIT} rebase --abort
+            #{GIT} checkout -f master
+            #{GIT} reset --hard origin/master
```

Comment:
I think these can probably stay as-is but up to you.

Reply:
I'm fine either way.

For some reason though it doesn't show as `/usr/bin/git` in GitHub actions and I don't quite get why.

Reply:
Well spotted. Could it be it's not testing the right revision?

Reply:
It's weird since the previous step used it fine. Lets see if it happens again I guess.

Reply:
Or perhaps it's pulling it from master at some point?

Reply:
I see the issue. I'll try fix it.